cmd/analyze/flag/flag.go

```diff
@@ -0,0 +1,95 @@
+/*
+Copyright © 2022 Flagship Team flagship@abtasty.com
+
+*/
+package flag
+
+import (
+	"encoding/json"
+	"log"
+	"os"
+
+	cbaConfig "github.com/flagship-io/codebase-analyzer/pkg/config"
+	"github.com/flagship-io/codebase-analyzer/pkg/handler"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+var (
+	Directory           string
+	RepoURL             string
+	RepoBranch          string
+	NbLineCodeEdges     int
+	FilesToExcludes     string
+	SearchCustomRegex   string
+	CustomRegexJsonFile string
+	CustomRegexJson     string
+)
+var FSConfig *cbaConfig.Config
+
+// FlagCmd represents the flag command
+var FlagCmd = &cobra.Command{
+	Use:   "flag [create|list]",
+	Short: "Analyze your codebase and detect the usage of Flagship or custom flags",
+	Long:  `Analyze your codebase and detect the usage of Flagship or custom flags, in order to synchronize them with your Flag view in the platform`,
+	PreRun: func(cmd *cobra.Command, args []string) {
+		var filesToExcludes_ []string
```

Comment:
typo

Reply:
why adding _ after the name of variable ? maybe it can be not easy to understand